### ISSUE TITLE
Search for a user

### ISSUE THREAD

**Issue by hueper** [rbytr/rbytr#18]
Users should be able to search for other users

**Comment by DanielPerlman** [comment]
Where should they access the search, in the side menu? It could be full-width text input on the top of the menu bar

**Comment by hueper** [comment]
[Kickstarter](http://kickstarter.com) offers a convenient way of searching: 
- User clicks on magnifier
- Input pops out on top
- Incremental search: results show up as user types

WDYT?

**Comment by DanielPerlman** [comment]
Wouldn't be too difficult to implement, the results could also show up as an overlay instead of a new section the way kickstarter does it?

**Comment by hueper** [comment]
Yes good idea, let's reuse the [transition](https://github.com/rbytr/rbytr/blob/master/modules/core/client/css/core.css#L1125) from opening the side menu and display results above.
